yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/DockerCloud.java

```diff
@@ -258,6 +264,21 @@ private DockerSlave provisionWithWait(DockerSlaveTemplate template, Provisioning
         return new DockerSlave(slaveName, nodeDescription, launcher, containerId, template, getDisplayName(), id);
     }
 
+    /** Determine the operating system associated with an image. */
+    private OsType determineOsType(String imageId) {
+        InspectImageResponse ir = getClient().inspectImageCmd(imageId).exec();
+        if ("linux".equals(ir.getOs())) {
```

Comment:
On my test systems, I see `"windows"` and `"linux"`.  However, I can change it to `equalsIgnoreCase` to be safe.